modules/data-streams/src/yamlRestTest/resources/rest-api-spec/test/data_stream/170_modify_data_stream.yml

```diff
@@ -116,6 +123,23 @@
         name: data-stream-for-modification2
   - is_true: acknowledged
 
+  # Initialize failure store
```

Comment:
Can we add a test case for modifying a data stream's failure store while it has no indices as well?